modules/openapi-generator/src/main/resources/cpp-qt-client/api-body.mustache

```diff
@@ -790,7 +790,7 @@ void {{classname}}::{{nickname}}({{#allParams}}{{#required}}const {{{dataType}}}
 
     connect(_latestWorker, &{{prefix}}HttpRequestWorker::on_execution_finished, this, &{{classname}}::{{nickname}}Callback);
     connect(this, &{{classname}}::abortRequestsSignal, _latestWorker, &QObject::deleteLater);
-    connect(_latestWorker, &QObject::destroyed, [this](){
+    connect(_latestWorker, &QObject::destroyed, this, [this](){
```

Comment:
```suggestion
    connect(_latestWorker, &QObject::destroyed, this, [this] {
```

Reply:
@Jazzco has this feedback been reviewed and incorporated into your change?

Reply:
Suggestion applied